shell/run_tests.sh

```diff
@@ -9,4 +9,11 @@ fi
 export PYTHONPATH=${HOMEROOT}:${HOMEROOT}/lib:${HOMEROOT}/IDC-Common
 echo "PYTHONPATH IS ${PYTHONPATH}"
 
-python3 ./manage_unit_test.py test --noinput
\ No newline at end of file
+#python3 ./manage_unit_test.py test --noinput
+
+python3 ./manage_unit_test.py test solr_helpers google_helpers.bigquery --noinput
+#python3 ./manage_unit_test.py test  --noinput
+
+echo "Running module unit tests with unittest..."
+
+#python3 -m unittest solr_helpers
```

Comment:
Is there a reason this one is off for the moment?

Reply:
I first tried to do solr_helpers tests with unittest, but it did not work because __init__.py calls Django. In general, "python3 -m unittest" barfs because the tests.py files it finds are not written for unittest, but Django tests. So I just commented it out until there are actual unittest tests to enumerate.

Reply:
Got it.